### ISSUE TITLE
Issue when  converting Tensorflow model, 'NoneType' object has no attribute 'run'

### ISSUE THREAD

**Issue by seranus** [mil-tokyo/webdnn#830]
Hi, I was following the conversion tutorial and I'm not sure if I'm doing it right, I'd appreciate if you could take a look

Trying to convert this model https://github.com/YadiraF/PRNet

Here's my code:
```
  TensorFlowConverter().convert([x], [pos])
  exec_info = generate_descriptor("webgpu", graph)
  exec_info.save("./output")
```

Added it there: https://github.com/YadiraF/PRNet/blob/master/predictor.py#L72

After I execute the code I get 'NoneType' object has no attribute 'run
Both prams are tensors

**Comment by milhidaka** [comment]
If inserting to line 72, session is not yet initialized.

Instead, inserting the code above line 97 will be correct.

```
graph = TensorFlowConverter(self.sess).convert([self.x], [self.x_op])
exec_info = generate_descriptor("webgpu", graph)
exec_info.save("./output")
```

https://github.com/YadiraF/PRNet/blob/233754d400d9239acea001c0aa0fe9172765b940/predictor.py#L97

However, there is another error.
```
Traceback (most recent call last):
  File "run_basics.py", line 37, in <module>
    pos = prn.process(image, kpt) # kpt information is only used for detecting face and cropping image
  File "/Users/hidaka/miljs/mil-web-dnn/PRNet/api.py", line 119, in process
    cropped_pos = self.net_forward(cropped_image)
  File "/Users/hidaka/miljs/mil-web-dnn/PRNet/api.py", line 61, in net_forward
    return self.pos_predictor.predict(image)
  File "/Users/hidaka/miljs/mil-web-dnn/PRNet/predictor.py", line 102, in predict
    graph = TensorFlowConverter(self.sess).convert([self.x], [self.x_op])
  File "/Users/hidaka/miljs/mil-web-dnn/webdnn/src/graph_transpiler/webdnn/frontend/tensorflow/converter.py", line 96, in convert
    self._convert_operator(op)
  File "/Users/hidaka/miljs/mil-web-dnn/webdnn/src/graph_transpiler/webdnn/frontend/converter.py", line 117, in _convert_operator
    self._handler_map[self.__class__.__name__][operator_key](self, operator)
  File "/Users/hidaka/miljs/mil-web-dnn/webdnn/src/graph_transpiler/webdnn/frontend/tensorflow/ops/gen_array_ops.py", line 644, in strided_slice_handler
    y = x[slices]
  File "/Users/hidaka/miljs/mil-web-dnn/webdnn/src/graph_transpiler/webdnn/graph/variable.py", line 321, in __getitem__
    return webdnn.graph.operators.slice.Slice(None, indices=indices)(self)[0]
  File "/Users/hidaka/miljs/mil-web-dnn/webdnn/src/graph_transpiler/webdnn/graph/operators/slice.py", line 113, in __call__
    """)
NotImplementedError: 
[Slice] Accessing to one element is not supported:
    (indices) = {?357:2}
```
This seems to be caused by conv2d_transpose.

**Comment by seranus** [comment]
Thank you for helping

I would suggest you add more documentation on custom operators and add some more basic operators

I tried to port multiple models they mostly needed something to be implemented other couldn't allocate a WebGl texture like mentioned in one of the issues. In the end none of the models i tried worked.

I looked to implement UpSamplig2d for one of my models, After doing some research couldn't find a combination of basic operators to do so, something like slice one element + concatenate.  Then I looked at the basic operators and without a documentation or a cookbook how are the webgl opertors implemented I gave up.

It feels like that at the current state there's no point in trying to port any real models, sorry for the rant.
